test/functionalTest/cases/2750_common_metrics/2750_incomingTransactions.test

```diff
@@ -0,0 +1,200 @@
+#
+
+echo "01. Send five GET /version"
+echo "=========================="
+orionCurl --url /version --tenant S1 --servicePath /SP1
+echo
+orionCurl --url /version --tenant S1 --servicePath /SP1
+echo
+orionCurl --url /version --tenant S1 --servicePath /SP2
+echo
+orionCurl --url /version --tenant S2 --servicePath /SP3
+echo
+orionCurl --url /version --tenant S3 --servicePath /SP4
+echo
+echo
+
+
+echo "02. Ask for Metrics, see 6 Incoming Transactions (the GET metrics request also counts)"
+echo "======================================================================================"
+orionCurl --url /admin/metrics
+echo
+echo
+
+
+--REGEXPECT--
+01. Send five GET /version
+==========================
+HTTP/1.1 200 OK
+Content-Length: REGEX(.*)
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "orion": {
+        "compile_time": "REGEX(.*)",
+        "compiled_by": "REGEX(.*)",
+        "compiled_in": "REGEX(.*)",
+        "git_hash": "REGEX(.*)",
+        "uptime": "REGEX(.*)",
+        "version": "REGEX(.*)"
+    }
+}
+
+HTTP/1.1 200 OK
+Content-Length: REGEX(.*)
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "orion": {
+        "compile_time": "REGEX(.*)",
+        "compiled_by": "REGEX(.*)",
+        "compiled_in": "REGEX(.*)",
+        "git_hash": "REGEX(.*)",
+        "uptime": "REGEX(.*)",
+        "version": "REGEX(.*)"
+    }
+}
+
+HTTP/1.1 200 OK
+Content-Length: REGEX(.*)
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "orion": {
+        "compile_time": "REGEX(.*)",
+        "compiled_by": "REGEX(.*)",
+        "compiled_in": "REGEX(.*)",
+        "git_hash": "REGEX(.*)",
+        "uptime": "REGEX(.*)",
+        "version": "REGEX(.*)"
+    }
+}
+
+HTTP/1.1 200 OK
+Content-Length: REGEX(.*)
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "orion": {
+        "compile_time": "REGEX(.*)",
+        "compiled_by": "REGEX(.*)",
+        "compiled_in": "REGEX(.*)",
+        "git_hash": "REGEX(.*)",
+        "uptime": "REGEX(.*)",
+        "version": "REGEX(.*)"
+    }
+}
+
+HTTP/1.1 200 OK
+Content-Length: REGEX(.*)
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "orion": {
+        "compile_time": "REGEX(.*)",
+        "compiled_by": "REGEX(.*)",
+        "compiled_in": "REGEX(.*)",
+        "git_hash": "REGEX(.*)",
+        "uptime": "REGEX(.*)",
+        "version": "REGEX(.*)"
+    }
+}
+
+
+02. Ask for Metrics, see 6 Incoming Transactions (the GET metrics request also counts)
+======================================================================================
+HTTP/1.1 200 OK
+Content-Length: 343
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "services": {
+        "": {
```

Comment:
Funny...

Maybe this is due to the `/admin/metrics` transaction?

Btw, not sure how to deal with the default service... Desing document says:

> There is a special key case, which actually don't correspond to a real service: **.default** corresponds to the default service (only used by CB).

However, it has been suggested to use another alternative. Maybe this is not stuff for this PR, but at least add the item in the list in the MetricsMgr.h comment.